packages/material-ui-lab/src/DataGrid/DataGrid.js

```diff
@@ -176,6 +181,13 @@ const DataGrid = React.forwardRef(function DataGrid(props, ref) {
     defaultSorting = [],
     loading = false,
     onSortingChange,
+    pagination = true,
+    paginationPage = 0,
```

Comment:
I think that we should leave the active page uncontrolled by default.

```suggestion
    paginationPage,
```

Reply:
This prop will be passed as an initial page to the pagination state, so it should be 0 so that the initial page is 0 if nothing is provided.

Reply:
Ok, in this case, we can implement the default page 0 later down the control flow. I think that we should look for `paginationPage != null` to determine if the current page is controlled or not.

Reply:
It's similar to the sorting prop.

Reply:
Yes looks good, I will change it to that

Reply:
What do you say to change it to initialPage and initialPageSize, so the user can navigate the table but the programmer can change the initial page e.g. last page.